ui-admin/src/populate/PopulateRouteSelect.tsx

```diff
@@ -4,28 +4,40 @@ import PopulatePortalControl from './PopulatePortalControl'
 import PopulateSurveyControl from './PopulateSurveyControl'
 import PopulateSiteContentControl from './PopulateSiteContent'
 import PopulateAdminConfig from './PopulateAdminConfig'
+import { navDivStyle, navListItemStyle } from 'util/subNavStyles'
+import { renderPageHeader } from 'util/pageUtils'
 
 /** shows links to the populate control panels, and handles the routing for them */
 export default function PopulateRouteSelect({ portalShortcode }: {portalShortcode?: string}) {
   /** styles links as bold if they are the current path */
-  const navStyleFunc = ({ isActive }: {isActive: boolean}) => {
-    return isActive ? { fontWeight: 'bold' } : {}
+  function getLinkCssClasses({ isActive }: { isActive: boolean }) {
+    return `${isActive ? 'fw-bold' : ''} d-flex align-items-center`
   }
 
   return <div className="container-fluid">
-    <div className="bg-white p-3 ps-5 row">
-      <div className="col-md-2 px-0 py-3 mh-100 bg-white border-end">
-        <h2 className="h4 px-3">Populate</h2>
-        <ul className="">
-          <li className="py-1"><NavLink to="portal" style={navStyleFunc}>Portal</NavLink></li>
-          <li className="py-1"><NavLink to="survey" style={navStyleFunc}>Survey</NavLink></li>
-          <li className="py-1">
-            <NavLink to="siteContent" style={navStyleFunc}>Site Content</NavLink>
-          </li>
-          <li className="py-1"><NavLink to="adminConfig" style={navStyleFunc}>Admin config</NavLink></li>
-        </ul>
+    { renderPageHeader('Populate') }
+    <div className="d-flex">
+      <div className="bg-white p-3 ps-5 row">
+        <div className="col-md-2 px-0 py-3 mh-100 bg-white">
+          <div style={navDivStyle}>
```

Comment:
I think since you imported that navDivStyle, you can drop all the bootstrap row and col-md-2 styles above (and possibly drop those divs altogether) -- the navDivStyle handles the width

Reply:
Done